### ISSUE TITLE
Invalid MEX-file

### ISSUE THREAD

**Issue by T34gr** [arokem/python-matlab-bridge#188]
Hi!

I tried to run in python 2.7.5+ the suggested code to check if it works and happens the same as to https://github.com/arokem/python-matlab-bridge/issues/186

My OS is Linux 3.11.0-12-generic x86_64 (linux mint 16)

I installed with pip pyzmq, Numpy, pymatbridge

I know that this issue is stated before but i am new both in linux and python and maybe the problem that occurs could be my mistake.

Any suggestions?

Above all thanks for your time trying to develop python-matlab bridge.

>>> from pymatbridge import Matlab
>>> mlab = Matlab()
>>> mlab.start()
Starting MATLAB on ZMQ socket ipc:///tmp/pymatbridge
Send 'exit' command to kill the server
resultsWarning: No window system found.  Java option 'MWT' ignored

                            < M A T L A B (R) >
                  Copyright 1984-2012 The MathWorks, Inc.
                    R2012a (7.14.0.739) 64-bit (glnxa64)
                              February 9, 2012

 
To get started, type one of these: helpwin, helpdesk, or demo.
For product information, visit www.mathworks.com.
 
Invalid MEX-file
'/usr/local/lib/python2.7/dist-packages/pymatbridge/matlab/messenger.mexa64':
libzmq.so.3: cannot open shared object file: No such file or directory

Error in matlabserver (line 7)
messenger('init', socket_address);

>> ............................................................MATLAB session timed out after 60 seconds
MATLAB failed to start
False

**Comment by arokem** [comment]
Yeah - this should (hopefully) be fixed with the upcoming release.
Hopefully already next week.

**Comment by arokem** [comment]
The new release is now out. Would you please try installing this again (you can use a pip upgrade, I believe) and reporting back?
